yaksh/documentation/moderator_docs/creating_question.rst

```diff
@@ -19,23 +19,33 @@ Setting up questions
 
 	* **Language** - Programming language on which the question is based.
 
-	* **Active** -  If the question is active to attempt or not.
```

Comment:
@prabhuramachandran Small correction, this field is no longer available to the User. What @adityacp is correct.